app/src/main/java/wordtextcounter/details/main/feature/input/InputViewModel.kt

```diff
@@ -6,45 +6,50 @@ import wordtextcounter.details.main.util.Helper
 
 class InputViewModel : BaseViewModel() {
 
-    data class ViewState(
-            val showError: Boolean = false,
-            val errorMessage: String = "",
-            val noOfWords: String = "0",
-            val noOfCharacters: String = "0",
-            val noOfSentences: String = "0"
-    )
-
-    data class Report(
-            val value: String = "0",
-            val reportType: ReportType
-    )
-
-    data class ReportState(val list: List<Report>)
-
-    val viewState: MutableLiveData<ViewState> = MutableLiveData()
-
-    init {
-        viewState.value = ViewState()
+  data class ViewState(
+      val showError: Boolean = false,
+      val errorMessage: String = "",
+      val noOfWords: String = "0",
+      val noOfCharacters: String = "0",
+      val noOfSentences: String = "0",
+      val noOfParagraphs: String = "0",
+      val size: String = "0",
+      val reportText: String = "",
+      val showExpand: Boolean = false
```

Comment:
I'm going to wrap all these values into `Report` entity type and inject that into this ViewState as I feel that's gonna let me remove redundant converter. Thoughts?

Reply:
#30